src/config/config.cpp

```
@@ -540,3 +542,105 @@ void print_default_env() {
+  for (const auto& l : lines) {
+    out << l << "\n";
+  }
+  return true;
+}
+
+// --- Named host registry (ADR-108) ---
+
+#include "json/json.h"
+
+/// Check if model name indicates a small model (≤7B) needing simpler prompt (ADR-109)
+bool is_small_model(const std::string& model_name) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)